lte/gateway/c/oai/tasks/nas/emm/Attach.c

```diff
@@ -710,6 +710,18 @@ int emm_proc_attach_complete(
           (nas_emm_attach_proc_t*)
               ue_mm_context->emm_context.emm_procedures->emm_specific_proc;
 
+      /* Process attach complete msg only if T3450 timer is running
+       * If its not running it means that implicit detach is in progress
+       */
+      if (attach_proc->T3450.id == NAS_TIMER_INACTIVE_ID) {
+        OAILOG_WARNING_UE(
+            LOG_NAS_EMM, ue_mm_context->emm_context._imsi64,
+            "Discarding attach complete as T3450 timer is not active for "
+            "ueid " MME_UE_S1AP_ID_FMT "\n",
+            ue_id);
+        bdestroy((bstring)(esm_msg_pP));
```

Comment:
isn't this a violation of constness of this parameter? Shall we instead return error and let caller destroy it?

Reply:
fixed